src/nouislider.core.less

```diff
@@ -9,7 +9,7 @@
 -webkit-tap-highlight-color: rgba(0,0,0,0);
 -webkit-user-select: none;
 -ms-touch-action: none;
-	touch-action: none;
+	touch-action: pan-y;
```

Comment:
Why are you changing touch action to `pan-y` here?

Reply:
Any update on this?

Reply:
Please discuss this in #788